### PR TITLE
Prefer using `httr2` and `V8` over `chromote`

### PR DESCRIPTION
**Pull request by jdhoffa**
This PR does several things...

We were getting timeout and response errors using `chromote` when trying to run `get_index_regions` on the VM. We weren't sure what was going on, but it seemed to be an issue with waiting for `loadEventFired()`. 

@AlexAxthelm suggested we try to not depend on `chromote` in general, because it's a bit bulky and probably overkill for our purposes. 

To that end, I tried to replace the `chromote` implementation with a more lightweight `httr2` implementation, and scrape all of the data using standard HTTP requests. This posed an issue as well, since the data that we want for Emerging Markets countries doesn't actually exist initially in the HTML, but is parsed from a JSON object and rendered in JS when the page is loaded in the browser. 

To get around this, I used `V8` which is a minimal and lightweight R interface for Google's open-source JavaScript engine. This allowed me to run the JS code and parse the resulting JSON, which I then used to pull the relevant countries. 

# tl;dr
* Lost `chromote` in favour of the lighter-weight `V8`

### CODE REVIEW

File: R/get_index_regions.R [hunk 1]
```diff
@@ -27,15 +26,32 @@ get_index_regions <- function() {
...
-    rvest::html_text2() %>%
-    strsplit("\n") %>%
-    unlist()
+  js_scripts <- rvest::html_elements(em_html, "div.portlet-body script")
+
+  js_script <- js_scripts[grepl("chartDataEMJson", js_scripts)]
+
+  ct <- V8::v8()
+  #parse the html content from the js output and print it as text
+  ct$eval(gsub('document.write', '', rvest::html_text2(js_script)))
```
Comment (AlexAxthelm) [inline R/get_index_regions.R:37]: NB: consider `str_replace` over `gsub` here

Comment (AlexAxthelm) [inline R/get_index_regions.R:37]: NB: Also consider doing the script modification into a separate variable, for clarity.
```suggestion
    render_script <- str_replace(
      pattern = 'document.write',
      replacement = '',
      string = rvest::html_text2(js_script)
    )
    ct$eval(render_script)
```

**Review by AlexAxthelm — APPROVED** [review]
